docs/source/deployment_guide_overview.md

```diff
@@ -0,0 +1,151 @@
+
+However and wherever your components are deployed, you will need a high degree of expertise in your management system of choice (such as Kubernetes) in order to efficiently operate your network. Similarly, the structure of the network must be designed to fit the business use case and any relevant laws and regulations government of the industry where the network will be designed to function.
+
+This deployment guide will not go through every iteration and potential network configuration, but does give common guidelines and rules to consider.
+
+## Step two: Set up a cluster for your resources
+
+Generally speaking, Fabric is agnostic to the method used to deploy and manage it. It is possible, for example, to deploy and manage a peer from a laptop. For a number of reasons, this is likely to be unadvisable, but there is nothing in Fabric to prohibit it.
+
+As long as you have the ability to deploy Docker containers, whether locally (or behind a firewall), or in a cloud, it should be possible to stand up components and connect them to each other. However, Kubernetes features a number of helpful tools that have made it a popular container management platform for deploying and managing Fabric networks. For more information about Kubernetes, check out [the Kubernetes documentation](https://kubernetes.io/docs).
+
+However and wherever you choose to deploy your components, you will need to make sure you have enough resources for the components to run effectively. The sizes you need will largely depend on your use case. If you plan to join a single peer to several high volume channels, it will need much more CPU and memory than a peer a user plans to join to a single channel. As a rough estimate, plan to dedicate approximately three times the resources to a peer as you plan to allocate to a single ordering node (as you will see below, it is recommended to deploy at least three and optimally five nodes in an ordering service). Similarly, you should need approximately a tenth of the resources for a CA as you will for a peer. By deploying a proof of concept network and testing it under load, you will have a better sense of the resources you will require.
+
+### Managing your infrastructure
+
+The exact methods and tools you use to manage your backend will depend on the backend you choose. However, here are some considerations worth noting if you choose some of the more popular deployment methods (Docker containers running in a Kubernetes cluster).
+
```

Comment:
Would they use the secrets to store the crypto material as well?

Reply:
I'm actually not sure.

Reply:
There are a number of options to consider here with regard to your question Nik, and HSMs, encrypted Persistent Volumes (PV's) in Kubernetes (k8s) are just a few to choose from.  The management piece Joe references here is more related to k8s secrets for managing 'pulls' from private container registries that are referenced in k8s deployment yaml files among other things.  When referencing images that are located in an open repo (like Fabric), secrets are not required.  After I deploy my Fabric components and want to connect my backend code container however, it is likely that I am using a private repo in say Docker Hub and I need to code the login information in the form of a k8s secret and include it in the yaml file for the deployment.